hyperspy/_signals/signal2d.py

```diff
@@ -732,6 +740,116 @@ def add_ramp(self, ramp_x, ramp_y, offset=0):
         ramp += ramp_y * yy
         self.data += ramp
 
+    def find_peaks2D(self, method='local_max', interactive=True,
+                     current_index=False, show_progressbar=None,
+                     parallel=None,  display=True, toolkit=None, **kwargs):
+        """Find peaks in a 2D signal.
+
+        Function to locate the positive peaks in an image using various, user
+        specified, methods. Returns a structured array containing the peak
+        positions.
+
+        Parameters
+        ----------
+        method : str
+             Select peak finding algorithm to implement. Available methods
+             are:
+                 'local_max' - simple local maximum search using the
+                 `scikit-image` `peaks_local_max` function.
+
+                 'max' - simple local maximum search - call the peak finder
+                 implemented in `scikit-image` which uses a maximum filter
+
+                 'minmax' - finds peaks by comparing maximum filter results
+                 with minimum filter, calculates centers of mass
+
+                 'zaefferer' - based on gradient thresholding and refinement
+                 by local region of interest optimisation
+
+                 'stat' - statistical approach requiring no free params.
+                 'massiel' - finds peaks in each direction and compares the
+                 positions where these coincide.
+
+                 'laplacian_of_gaussian' - a blob finder implemented in
+                 `scikit-image` which uses the laplacian of Gaussian
+                 matrices approach.
+
+                 'difference_of_gaussian' - a blob finder implemented in
+                 `scikit-image` which uses the difference of Gaussian
+                 matrices approach.
+
+                 xc' - A cross correlation peakfinder
```

Comment:
missing "'" - and is there any more detail you can give on what it does/where it comes from?